networking/external_dns_operator/external-dns-operator-release-notes.adoc

```diff
@@ -0,0 +1,68 @@
+// External DNS Operator Release Notes
```

Comment:
Really minor nit, but I would remove this commented-out line so that the content type line is on line 1.

Reply:
Thank you